vendor/processors/stripe_processor.py

```diff
@@ -451,11 +452,26 @@ def build_setup_intent(self, payment_method_id):
     def build_subscription(self, subscription, payment_method_id):
         return {
             'customer': self.invoice.profile.meta['stripe_id'],
+            'coupon': subscription.offer.meta['stripe'].get('coupon_id'),
```

Comment:
What if no 'coupin_id' is there. Can the API handle `coupon = None`?

Reply:
It does matter if you add a key and set it to None. Stripe API allows it

Reply:
cool looks good